ARE_RestAPIlibraries/clientExample/javascript/areCommunicator.js

```diff
@@ -1,4 +1,3 @@
-
 //The base URI that ARE runs at
```

Comment:
What is the plan for `areCommunicator.js` for the future? Do we want to keep it and maintain it here? Or with `npm`? Or both?

Reply:
Let's keep it here to have a consistent version (being complete and compatible) for this release.
But in future we will only rely on the new npm-released version and skip the others step by step.

Reply:
theoretically it should be possible to generate the `areCommunicator.js` out of the existing REST methods defined in the Java classes, shouldn't it? All information needed to create it is included there. The same could be the case for the Java REST client classes.